### DISCUSSION TITLE
Diagramme für Leistung und Energie

### DISCUSSION THREAD

**Discussion by chef76** [brutella/hkknx-public#318] [Ideas]
Im Dashboard kann man das ja schon anzeigen, aber es gibts ja noch keine HomeKit Gerätekategorie für Zähler. Mit dem Stop für Jalousien hast du dich ja schon von HomeKit gelöst, vielleicht könnte man auch  Geräte für Leistungs- und Energiemesswerte anlegen?!?

![image](https://github.com/brutella/hkknx-public/assets/69462419/ebd3421b-e7d3-42cb-ba1d-feceba98edb1)

**Comment by brutella** [comment]
> Mit dem Stop für Jalousien hast du dich ja schon von HomeKit gelöst [...]

Das Stop für Jalousien ist mit HomeKit kompatibel, nur leider unterstützt die Apple Home-App und Siri die Funktion nicht. Also strengt genommen als HomeKit-konform. 😉

> [...] vielleicht könnte man auch Geräte für Leistungs- und Energiemesswerte anlegen?!?

Ich kann mir gut vorstellen, Geräte zu unterstützen, die nicht HomeKit-kompatibel sind. Als Beispiele fallen mir folgende Gerätetypen ein.

- Wettersensoren: Wind, Regen
- Energiemanagement: Stromzähler, Wallboxen, Batterie

Gibt es Ideen für weitere Geräte?

Die Geräte würde man, wie gewohnt, mit hkknx konfigurieren. Die aktuellen und vergangene Werte kann man über die Weboberfläche ansehen. Mit HomeKit werden diese Geräte nicht geteilt, da diese nicht unterstützt werden.

**Reply by chef76** [reply]
Das hört sich doch schon ziemlich gut an. Die Liste kann ja mit der Zeit noch wachsen.

**Comment by brutella** [comment]
Mit Version 3 werden Werte von Gruppenadressen in der Web-Suche in Diagrammen dargestellt. Somit sollte das Problem gelöst sein.
